emr.install

```diff
@@ -20,3 +53,72 @@ function emr_update_8001(&$sandbox) {
   \Drupal::entityDefinitionUpdateManager()
     ->installFieldStorageDefinition('emr_default_revision', 'entity_meta', 'emr', $field);
 }
+
+/**
+ * Create a 'entity_meta_default_revision' table for default revision tracking.
+ */
+function emr_update_8002(&$sandbox) {
+  // Create the table.
+  $schema = [
+    'description' => 'Keeps track of the default entity meta revision.',
+    'fields' => [
+      'entity_meta_id' => [
+        'description' => 'The entity meta identifier.',
+        'type' => 'int',
+        'unsigned' => TRUE,
+        'not null' => TRUE,
+      ],
+      'default_revision_id' => [
+        'description' => 'The revision ID that is the default one.',
+        'type' => 'int',
+        'not null' => TRUE,
+      ],
+    ],
+    'indexes' => [
+      'default_revision' => [
+        'default_revision_id',
+        'entity_meta_id',
+      ],
+    ],
+    'primary key' => ['entity_meta_id'],
+  ];
+
+  \Drupal::database()->schema()->createTable('entity_meta_default_revision', $schema);
```

Comment:
We need to move the table creation to a separate hook_update_N() as under this we are creating batches and this fails when more than one batch exists.